core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -2651,6 +2651,16 @@ public String toString() {
       .setConsistencyCheckLevel(ConsistencyCheckLevel.ENFORCE)
       .setScope(Scope.WORKER)
       .build();
+  public static final PropertyKey WORKER_MASTER_PERIODICAL_CONNECT_TIMEOUT =
+      new Builder(Name.WORKER_MASTER_PERIODICAL_CONNECT_TIMEOUT)
+          .setDefaultValue("5min")
+          .setDescription("Timeout for periodical connections between workers "
+              + "and the leading master. Periodical connections between workers "
+              + "and previous leading master may hang which prevents workers "
+              + "from registering with the new leading master "
+              + "during flaky network situations.")
```

Comment:
Can we explain in description how the value of the property can affect the delay before a worker attempt to register with a new primary master?